### ISSUE TITLE
Support the system-wide text size in iOS

### ISSUE THREAD

**Issue by mikix** [standardnotes/forum#3826]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes.

**Is your feature request related to a problem? Please describe.**
When I have adjusted the system text size (Text Size in Settings, called the Dynamic Type feature in the dev docs I think: https://developer.apple.com/documentation/uikit/scaling-fonts-automatically ), most apps correctly increase their UI font accordingly.

But Standard Notes does not - it’s fonts remain small.

**Describe the solution you'd like**
Use the appropriate iOS API to dynamically set font size for both UI elements and note text.

**Comment by holmbuar** [comment]
Agreed, UI fonts and icons are too small, deviating from the iOS user experience, not the least the native Notes app.
